app/common/state/siteState.js

```diff
@@ -0,0 +1,18 @@
+const assert = require('assert')
+const { makeImmutable, isMap, isList } = require('./immutableUtil')
```

Comment:
nitpick- I don't think it's documented anywhere, but we don't have spaces when doing destructuring in most of our code.  So it could look like this instead:
`const {makeImmutable, isMap, isList} = require('./immutableUtil')`